rules_java_gapic/java_gapic_spring.bzl

```diff
@@ -0,0 +1,141 @@
+# Copyright 2022 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+load("@rules_gapic//:gapic.bzl", "proto_custom_library")
+
+def _java_gapic_spring_postprocess_srcjar_impl(ctx):
+    gapic_srcjar = ctx.file.gapic_srcjar
+    output_srcjar_name = ctx.label.name
+    srcjar_name = output_srcjar_name + "_raw.srcjar"
+    output_spring = ctx.outputs.spring
+    formatter = ctx.executable.formatter
+
+    output_dir_name = ctx.label.name
+    output_dir_path = "%s/%s" % (output_spring.dirname, output_dir_name)
+
+    script = """
+    WORKING_DIR=`pwd`
+    cd $WORKING_DIR
+    mkdir {output_dir_path}/spring
```

Comment:
I get below log printed when local testing with `vision`, believe it's from this line and L34 below. Do we still need `/spring` here?
`mkdir: cannot create directory 'bazel-out/k8-fastbuild/bin/google/cloud/vision/v1/vision_java_gapic_spring/spring': No such file or directory
find: 'bazel-out/k8-fastbuild/bin/google/cloud/vision/v1/vision_java_gapic_spring/spring': No such file or directory
`